### DISCUSSION TITLE
plain yaml in /var/lib/rancher/rke2/server/manifests is deleted

### DISCUSSION THREAD

**Discussion by vrbgtrhr** [rancher/rke2#7287] [Q&A] [answered]
im pasting in /var/lib/rancher/rke2/server/manifests an already templated helm but the file gets deleted and not applyed as it should

**Comment by dereknola** [comment]
Are you following the naming file requirements https://docs.rke2.io/helm#file-naming-requirements? What is the name of the file and what are its contents? Are you intending to replace an existing RKE2 component? Please provide more information into the issue.

**Comment by brandond** [comment, marked as answer]
What is your filename? I would avoid naming your file with an `rke2-` prefix, as this prefix is used by our internal components. Even if disabled, the component chart is written to that directory on startup, and then removed if disabled.
